src/main/java/pl/confitura/jelatyna/email/EmailParams.java

```diff
@@ -0,0 +1,35 @@
+package pl.confitura.jelatyna.email;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EmailParams {
```

Comment:
I don't like this name. It doesn't say what is this made for.
I figured it is to personalize mail messages from template... so maybe something like EmailPersonalizationData ?